ledger/participant-integration-api/src/main/scala/platform/store/dao/ParametersTable.scala

```diff
@@ -88,4 +110,8 @@ private[dao] object ParametersTable {
       LedgerEndAndConfigurationParser.single
     )(connection)
 
+  case class LedgerEndUpdateError(expected: Offset)
+      extends RuntimeException(s"""Could not update ledger end.
+         |Previous ledger end does not match expected ${expected.toHexString}""".stripMargin)
```

Comment:
Log lines should better be in a single line.
```suggestion
      extends RuntimeException(s"Could not update ledger end. Previous ledger end does not match expected ${expected.toHexString}")
```